ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -657,6 +666,10 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
                 << errno
                 << "(" << strerror(errno) << ")"
                 << endl;
+            string errorMessage = ss.str();
+
+            s_callbacks.LogError(errorMessage.c_str());
+            goto CleanupAndReturn;
```

Comment:
Just to be explicit about how we're handling, set `result` here before going to `CleanupAndReturn`?

Reply:
@nickgra @jeschu1 I think the changes look good and ready to test.  @nickgra can you give them a try (current changes return failure, could you test out returning both failure and success)?